tools/shared/src/main/scala/org/scalajs/core/tools/javascript/Trees.scala

```diff
@@ -241,4 +290,53 @@ object Trees {
       body: Tree)(implicit val pos: Position) extends Tree
 
   case class Super()(implicit val pos: Position) extends Tree
+
+  // ECMAScript 6 modules
+
+  /** `import` statement, except namespace import.
+   *
+   *  This corresponds to the following syntax:
+   *  {{{
+   *  import { <binding1_1> as <binding1_2>, ..., <bindingN_1> as <bindingN_2> } from <from>
+   *  }}}
+   *  The `_1` parts of bindings are therefore the identifier names that are
+   *  imported, as specified in `export` clauses of the module. The `_2` parts
+   *  are the names under which they are imported in the current module.
+   *
+   *  Special cases:
+   *  - When `_1 == _2.name`, there is shorter syntax in ES, i.e.,
+   *    `import { binding } from 'from'`.
+   *  - When `_1 == "default"`, it is equivalent to a default import.
+   */
+  case class Import(bindings: List[(String, Ident)], from: StringLiteral)(
```

Comment:
So `_1` isn't an `Ident` because there are different naming rules I assume? If yes, why don't we need an `Export` (or `ExportIdent`) tree?